app/models/cohort_columns/date_added_to_cohort.rb

```diff
@@ -10,8 +10,12 @@ class DateAddedToCohort < ReadOnly
     attribute :translation_key, String, lazy: true, default: 'Date Added to Cohort'
     attribute :title, String, lazy: true, default: ->(model, _attr) { _(model.translation_key) }
 
-    def value(cohort_client)
-      cohort_client.created_at.to_date
+    def default_value?
+      true
+    end
+
+    def default_value(_client_id)
+      Date.today
```

Comment:
I don't understand how this is going to work.  Won't we need to set this in the controller whenever someone is added to the cohort if you persist it?  The original plan was to use `GrdaWarehouse::CohortClientChange` to populate the value, though potentially that will be slow and it will need to be persisted/cached.  Maybe `default_value?` is the key?  If it is, do we need to clear it when removing someone from the cohort?

Reply:
Yes, `default_value` is the magic here... When you add a client to a cohort, it gets processed by `AddCohortClientsJob`. Which either creates a new `CohortClient`, or resurrects an old one if it has been previously removed, and then iterates over the columns setting/updating any that are flagged as having a default value.

Reply:
Got it!  Thanks